testing/mockeryutils/mockeryutils.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2024 The Nephio Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mockeryutils
+
+import (
+	"github.com/stretchr/testify/mock"
+)
+
+func InitMocks(mocked *mock.Mock, mocks []MockHelper) {
+	for counter := range mocks {
```

Comment:
It might be a good idea to add a check for `mocked -== nil` and `panic`, this would capture any incorrect usage by someone new to using the tool.